guake/palettes.py

```diff
@@ -1047,3 +1052,19 @@
         "#CFCFCFCFCFCF:#D9D9E6E6F2F2:#19191F1F1D1D"
     ),
 }
+
+themes_folder = os.path.expandvars("$HOME/.config/guake/themes")
+
+if os.path.isdir(themes_folder):
+    theme_filenames = next(os.walk(themes_folder))[2]
+    for theme_filename in theme_filenames:
+        with open(os.path.join(themes_folder, theme_filename)) as theme_file:
+            try:
+                theme_to_load = json.load(theme_file)
+                PALETTES = {**PALETTES, **theme_to_load}
```

Comment:
This might be inefficient. I came across a similar instance where `a_list.apply` was faster than list spread, so it might also be true here. Maybe an explicit injection is better or something more pythonic.